src/uu/cp/src/cp.rs

```diff
@@ -1613,10 +1640,12 @@ fn copy_extended_attrs(source: &Path, dest: &Path) -> CopyResult<()> {
 }
 
 /// Copy the specified attributes from one path to another.
+#[allow(unused_variables)]
 pub(crate) fn copy_attributes(
     source: &Path,
     dest: &Path,
     attributes: &Attributes,
+    options: &Options,
```

Comment:
What's the reason for this parameter?

Reply:
carryover from the previous work, I might be able to remove it